_scripts/container-create

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+in_toolbox=/run/.containerenv
+
+if [ -f $in_toolbox ]; then
+  podman="flatpak-spawn --host env podman"
+else
+  podman="env podman"
+fi
```

Comment:
This should work on a base OS or inside a toolbox (which gets passed to the host).

Reply:
Hah, I do [something similar](https://piware.de/gitweb/?p=bin.git;a=commitdiff;h=791a81a30e65e93bd6b10ec7ec983ec8fe79a727) :+1: